### DISCUSSION TITLE
Trying to get fooocus to read pre downloaded models

### DISCUSSION THREAD

**Discussion by JojoBaPb** [lllyasviel/Fooocus#1525] [Q&A] [answered]
Hi, so I downloaded the "juggernautXL v6_RunDiffusion" model and pasted into the ...\Fooocus_win64_2-1-831_3\Fooocus\models\checkpoints path. 

But whenever I try to launch the application it doesn't register that the model is there and tries to download juggernautXL.
Have I maybe pasted the model in the wrong folder? How do I get it to read that the model is present without having to redownload it?

**Comment by Bratzmeister** [comment]
I'd also wish for a way to suppress the auto download of checkpoints. Also there is a v7 version of juggernautXL now

**Reply by xizzat** [reply]
just add this in your config.txt

`"default_model": "juggernautXL_v7Rundiffusion.safetensors",`
`"checkpoint_downloads": {
        "juggernautXL_v7Rundiffusion.safetensors": "https://civitai.com/api/download/models/240840?type=Model&format=SafeTensor&size=full&fp=fp16"
    },`

**Reply by Bratzmeister** [reply]
but won't this be overwritten with every update?

**Reply by xizzat** [reply]
u mean config.txt? nope

**Comment by xhoxye** [comment, marked as answer]
Check the name of the model and the storage path.
"juggernautXL v6_RunDiffusion?"The name is wrong, and the correct one should be this "juggernautXL_version6Rundiffusion.safetensors"

**Reply by JojoBaPb** [reply]
Omg, thank you! I had it saved as "juggernautXL_version6Rundiffusion_2.safetensors" the "_2" was the problem lol
